tests/CurrencyHelperTest.php

```diff
@@ -109,4 +109,24 @@ public function testGetSymbol(string $currencyCode, string $locale, string $expe
 
         $this->assertEquals($expected, $actual);
     }
+
+    /**
+     * Tests that it returns formatted value from cents.
+     */
+    public function testToFormatFromInt()
+    {
+        $config = $this->createMock(Repository::class);
+        $currencyHelper = new CurrencyHelper($config);
+        $countries = require __DIR__ . '/../src/config/countries.php';
+
+        $config->expects($this->once())
+            ->method('get')
+            ->with('countries')
+            ->willReturn($countries);
+
+        $expected = '£15.50';
+        $actual = $currencyHelper->toFormatFromInt('1550');
+
+        $this->assertEquals($expected, $actual);
```

Comment:
This could be much simpler if `CurrencyHelper` is a Facade and we assume the config is correct, which is exactly the same thing we do already. We load whatever we have in the config, assuming it's correct.

```php
$this->assertEquals('£15.50', CurrencyHelper::toFormatFromInt('1550'));
```

Even if you want to pretend the config is correct you could add a stub before the assertion:

```php
config()->set('countries.GB', [
    'locale' => 'en',
    'systemLocale' => 'en_GB.UTF-8',
]);
```

Facades leverage DI in the end but I apparently this is a whole debate that should happen somewhere else.